webui/react/src/pages/ExperimentList.tsx

```diff
@@ -62,6 +62,8 @@ import { ErrorLevel } from 'shared/utils/error';
 import { validateDetApiEnum, validateDetApiEnumList } from 'shared/utils/service';
 import { alphaNumericSorter } from 'shared/utils/sort';
 import { humanReadableBytes } from 'shared/utils/string';
+import { useAuth } from 'stores/auth';
+import { useEnsureUsersFetched, useUsers } from 'stores/users';
```

Comment:
these lines and `OldProjectDetails.tsx` got changed during a merge commit - can be safely removed